configurations/coding_plans.py

```diff
@@ -119,7 +119,43 @@ def clean_age_with_range_filter(text):
                        ],
                        ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET.get_code_with_match_value("leap kalobeyei s01e05"),
                        raw_field_fold_strategy=FoldStrategies.concatenate),
-    ]
+
+    CodingPlan( raw_field="s01_lessons_learnt_raw",
+                dataset_name="s01_kalobeyei_lessons_learnt",
+                time_field="s01_lessons_learnt_time",
```

Comment:
since we are treating this as RQAs this should be set to "sent_on"

Reply:
updated, thanks